cmd/deploy/deploy.go

```diff
@@ -416,7 +420,7 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 	return err
 }
 
-func setDeployOptionsValuesFromManifest(deployOptions *Options, cwd string) {
+func setDeployOptionsValuesFromManifest(ctx context.Context, deployOptions *Options, cwd string, c kubernetes.Interface) {
```

Comment:
Nit: I don't like the idea of this function having these new parameters. It's not important for this feature, but we could try to pass in the dependant services instead of a whole kubernetes interface